src/components/album/AlbumsList.js

```diff
@@ -84,16 +96,15 @@ const AlbumsList = () => {
 
   //call the function to make API request for search input value
   const handleSearch = () => {
-    fetchAlbums(searchType, searchTerm, limit);
+    fetchAlbums(searchType, searchTerm);
   };
 
   //clear search input and make an empty API call
   const handleClear = () => {
-    setSearchType('albumName');
+    //setSearchType('albumName');
     setSearchTerm('');
-    setLimit(1);
     setMessage('');
-    fetchAlbums('albumName', '', 0);
+    fetchAlbums('albumName', '');
   };
```

Comment:
This approach is fine, but we also want to make sure that we go back to page 1 when we've cleared the search so that we're getting the fresh/blank search term results. Changing current page will call `fetchAlbums` for us via the useEffect on line 69:

```javascript
  const handleClear = () => {
    setSearchTerm('');
    setMessage('');
    setCurrentPage(1);
  };
```

Reply:
I added these changes